richtextfx/src/main/java/org/fxmisc/richtext/CaretPath.java

```diff
@@ -0,0 +1,10 @@
+package org.fxmisc.richtext;
+
+import javafx.scene.shape.Path;
+
+/**
+ * A path which describes a selection shape in the Scene graph.
+ * 
```

Comment:
"selection" is written instead of what you actually meant: "caret."

Otherwise, everything looks good!